core/src/main/java/tc/oc/pgm/PGMConfig.java

```diff
@@ -466,6 +472,11 @@ public String getMapPoolFile() {
     return mapPoolFile;
   }
 
+  @Override
+  public String getIncludesDirectory() {
```

Comment:
```suggestion
  public @Nullable String getIncludesDirectory() {
```
The interface this implements also has it, should probably be propagated down